pkg/credentialprovider/plugins.go

```diff
@@ -44,6 +46,12 @@ func RegisterCredentialProvider(name string, provider DockerConfigProvider) {
 	providers[name] = provider
 }
 
+// LegacyCredentialProvidersDisabled checks if the legacy credential providers
+// have been disabled.
+func LegacyCredentialProvidersDisabled() bool {
```

Comment:
Just so it's clear this only applies to the cloud provider based plugins, we should probably name this something like `IsCloudCredentialProvidersDisabled`

Reply:
Renamed it to AreLegacyCloudCredentialProvidersDisabled